mon-pix/app/components/routes/campaigns/restricted/join-sup.js

```diff
@@ -29,9 +29,11 @@ class Validation {
 
 export default class JoinSup extends Component {
   @service store;
-  @tracked errorMessage;
+  @tracked errorForQuickFormMessage;
+  @tracked errorForFurtherInformationMessage;
```

Comment:
Petit suggestion de renommage : 

```suggestion
  @tracked errorMessageForQuickForm;
  @tracked errorMessageForFurtherInformation;
```

On pourrait faire ressortir plus de métier dans ces noms. Le quick form c'est celui avec le numéro étudiant et further information c'est pour s'enregistrer en surnuméraire ?

Si c'est bien le cas, on pourrait nommer ces variables : 

```suggestion
  @tracked errorMessageForStudentNumberForm;
  @tracked errorMessageForSupernumerayForm;
```

Reply:
la variable qui affiche le reste du formulaire s'appelle `displayFurtherInformationForm`, c'était pour rester cohérent

Reply:
Oui je propose de renommer l'ensemble des variables si ça vous semble plus précis. Sinon on peut laisser tel quel 😄

Reply:
done